Source/Core/Core/BootManager.cpp

```diff
@@ -181,6 +183,12 @@ bool BootCore(const std::string& _rFilename)
 			}
 		}
 
+		// Some NTSC GameCube games such as Baten Kaitos react strangely to language settings that would be invalid on an NTSC system
+		if (!StartUp.bWii && StartUp.bNTSC)
```

Comment:
The bWii check is unnecessary, because the code below only sets the GameCube language, which Wii games don't care about.